assembl/models/auth.py

```diff
@@ -685,6 +690,40 @@ def __init__(self, **kwargs):
 
         super(User, self).__init__(**kwargs)
 
+    @classmethod
+    def populate_db(cls, db=None):
+        default_config = config.get_config()
+        # retrieve the configured machines
+        machines_data = default_config.get('machine.machines', '')
+        if machines_data:
+            from ..auth.util import add_user
+            db = db or cls.default_db()
+            # retrieve existing machines
+            machines = db.query(cls).filter(
+                cls.is_machine == True  # noqa: E712
+                )
+            # retrieve machines ids
+            all_machines_ids = [machine.username.username for machine in machines]
```

Comment:
Please use the `username_p` property here. We may remove the Username class in the near future.